### ISSUE TITLE
user needs accessibility

### ISSUE THREAD

**Issue by rrustema** [petities/petitions.eu#437]
User KXA-62799-522 complains about accessibility. When using Safari the 'Readers' feature should be possible for him to be able to read the website. Could this be solved with just one or a few style sheets?

**Comment by markkocera** [comment]
The 'Reader View' in Safari is only visible when there is a certain structure (as far as I know a h1 to h6 followed by text) and a certain amount of text on the page. Some pages on the site show the icon, others do not, for instance:

Showing:
https://petities.nl/petitions/pianola-museum-moet-blijven?locale=nl
https://petities.nl/petitions/stop-onnodige-plastic-verpakkingen-in-de-supermarkt?locale=nl
https://petities.nl/updates/rathenau-instituut-petities-nl-levert-burger-meeste-op-als-inspraaktraject?locale=nl

Not showing:
https://petities.nl/petitions/stop-de-sluiting-van-brownies-downies-rotterdam?locale=nl
https://petities.nl/petitions/stop-het-dierenleed?locale=nl

Presumably because there simply isn't enough text on the page. Using the Reader View shortcut on these pages (cmd-shift-R) also has no effect.

Safari probably takes the whole page into account, the petition text on this page: https://petities.nl/petitions/pianola-museum-moet-blijven?locale=nl for instance does not seem longer, but the petition has news items at the bottom of the page.

Can it be solved?
I do not know.

**Comment by rrustema** [comment]
Thanks, the solution is to work on general, web standard, accessibility and not this particular feature that is used for the same purpose but is designed for comfortably reading 'articles' using a specific browser. This will be the feedback to the user.
